### ISSUE TITLE
Fix dependency on node 8.x, so it can be used on later version of node

### ISSUE THREAD

**Issue by hawkeye64** [hfg-gmuend/openmoji#251]
```bash
$ yarn add -D openmoji
yarn add v1.22.4
[1/4] Resolving packages...
[2/4] Fetching packages...
error openmoji@12.3.0: The engine "node" is incompatible with this module. Expected version "8.x". Got "12.14.0"
error Found incompatible module.
```
I am not sure what is causing this as I looked in package.json and did not see restriction

Do you guys generate this npm package? https://www.npmjs.com/package/openmoji

**Comment by carlinmack** [comment]
I think you have to do `yarn upgrade --ignore-engines`, we've fixed this for the next release which is coming soon :)

**Comment by hawkeye64** [comment]
In the meantime, to get around this, I just `yarn add https://github.com/hfg-gmuend/openmoji`

**Comment by b-g** [comment]
Fixed and published/released with 12.4, a simple `npm install openmoji` should do the trick. Closed.
